modules/update-mco-process.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/how-updates-work.adoc
+
+:_content-type: CONCEPT
+[id="mco-update-process_{context}"]
+= Machine Config Operator node updates
```

Comment:
@rioliu-rh could you please take a look at the mco update section? Thank you.

Reply:
s/Machine Config Operator node updates/Understanding how the Machine Config Operator updates nodes
Currently, it sounds like we are updating the Machine Config Operator node.